index.bs

```diff
@@ -549,6 +549,18 @@ interface WebTransport {
   Promise&lt;WebTransportSendStream&gt; createUnidirectionalStream();
   /* a ReadableStream of WebTransportReceiveStream objects */
   readonly attribute ReadableStream incomingUnidirectionalStreams;
+
+  /* if unset, default is 1 second */
+  attribute unsigned long? rateControlFeedbackMinIntervalMs;
+  /* fires when the value changes, but at most every rateControlFeedbackMinIntervalMs */
+  attribute EventHandler onratecontrolfeedback;
+  readonly attribute RateControlFeedback rateControlFeedback;
+};
+
+[Exposed=(Window,Worker), SecureContext]
+interface RateControlFeedback {
+  /* The rate you should send, on average, to avoid queuing */
+  readonly attribute unsigned long? averageTargetBps;
```

Comment:
bits or bytes?

over what period?

Reply:
Bits, and I made the period question clear in the text backing up the IDL.